commit 58791f4040839030e0e3cab68975082d69973b1d (refs/changes/61/4261/2)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2018-06-19T16:14:47-04:00 (1 year, 4 months ago)
    
    TRITON-523 upload vminfod logs to manta
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/etc/logsets.json b/etc/logsets.json
index 929fd32..3086781 100644
--- a/etc/logsets.json
+++ b/etc/logsets.json
@@ -146,6 +146,22 @@
       "global"
     ]
   },
+  {
+    "name": "vminfod_logs",
+    "search_dirs": [ "/var/log/vm" ],
+    "regex": "^/var/log/vm/vminfod_([0-9a-zA-Z-]+)_([0-9]+)-([0-9]+)-([0-9]+)T([0-9]+):([0-9]+):([0-9]+)\\.log$",
+    "manta_path": "/%u/stor/logs/%d/vminfod/#y/#m/#d/#H/$1.log",
+    "date_string": {
+      "y": "$2", "m": "$3", "d": "$4",
+      "H": "$5", "M": "$6", "S": "$7"
+    },
+    "date_adjustment": "-1H",
+    "debounce_time": 600,
+    "retain_time": 86400,
+    "zones": [
+      "global"
+    ]
+  },
   {
     "name": "sdcadm_logs",
     "search_dirs": [ "/var/log/sdcadm" ],
